src/components/HintIcon.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <app-icon
     name="question"
-    :size="19"
+    :size="16"
```

Comment:
If It does not break the alignment I would give a size of at least `18`, now It looks a bit too small.

Reply:
I think we might need different sizes based on where it's used. For example, a hint icon next to h1 should probably be bigger than in table headers.

Reply:
That's another improvement but for now I would simply make it a little bit bigger.

Reply:
Alright, I extended it to 17.
I took the inspiration from some other explorers where the icon is even smaller
![image](https://github.com/aeternity/aescan/assets/15363559/92c2b2fa-d065-44fe-971e-a5be0592e57e)

![image](https://github.com/aeternity/aescan/assets/15363559/3bfba520-95b6-4ef7-b9f7-57d18f3b2d7d)